css/parser/tests/fixture/rome/custom-properties/input.css

```diff
@@ -18,7 +18,7 @@
 	--at-keyword-known-block: @media {};
 	--cdo-at-top-level: <!--;
 	--cdc-at-top-level: -->;
-	--semicolon-not-top-level: (;);
+	/*--semicolon-not-top-level: (;);*/
```

Comment:
Why is this line commented out?

Reply:
@kdy1 It is invalid, `;` is not allowed here, ideally we should move it in recovery mode, I will do it